src/Upgrade/Infrastructure/Processor/Strategy/IntegratorClient/IntegratorClient.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrade\Infrastructure\Processor\Strategy\IntegratorClient;
+
+use Upgrade\Infrastructure\Dto\Step\StepsExecutionDto;
+use Upgrade\Infrastructure\Process\ProcessRunner;
+
+class IntegratorClient implements IntegratorClientInterface
+{
+    /**
+     * @var string
+     */
+    protected const RUNNER = '/home/spryker/.composer/vendor/bin/integrator';
```

Comment:
What will in case when integrator is not installed globally? 
Please use `vendor/bin/integrator ...`?

Reply:
Please find more proper naming for the current constant